### DISCUSSION TITLE
Exception on ShowDialog Method

### DISCUSSION THREAD

**Discussion by Progiscad** [MahApps/MahApps.Metro#4479] [General]
Hello,

I have a problem when I use the ShowDialog method on a new thread and only in Net 8, I do not have the problem in Net Standard.

 Can you help me please ?

I join a visual studio project net 8 join here :
[testmodal.zip](https://github.com/MahApps/MahApps.Metro/files/14980689/testmodal.zip)
Thank.

**Comment by punker76** [comment]
@Progiscad If I add the missing styles and resources to your sample everything is working: https://mahapps.com/docs/guides/quick-start#mahapps-build-in-styles-and-themes

![image](https://github.com/MahApps/MahApps.Metro/assets/658431/40ecb619-e371-4bc7-a132-d74d5c99ce37)

**Comment by Progiscad** [comment]
Thank you very much and sorry for this oversight
